### PR TITLE
Convert radix styles to tokens

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/2151 https://github.com/webstudio-is/webstudio-builder/issues/2137

Converted here everything except tabs. Will do them in separate PR.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
- [ ] hi @istarkov, I need you to do
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/sdk-components-react-radix/src/sheet.ws.tsx [hunk 1]
```diff
@@ -71,35 +70,135 @@ export const metaSheetContent: WsComponentMeta = {
...
+        tc.state(
+          [tc.mr("auto"), tc.maxW("sm"), tc.grow()].flat(),
+          "[data-side=left]"
+        ),
+        tc.state(
+          [tc.ml("auto"), tc.maxW("sm"), tc.grow()].flat(),
+          "[data-side=right]"
+        ),
+        tc.state([tc.mb("auto")].flat(), "[data-side=top]"),
+        tc.state([tc.mt("auto")].flat(), "[data-side=bottom]"),
```
Comment (istarkov) [inline packages/sdk-components-react-radix/src/sheet.ws.tsx:101]: Must be tokens, we just don't need this states and don't need bind data-side to side property

**Comment by kof** [comment]
@taylornowotny  check out the token names


Screenshot 2023-08-22 at 21 07 04
Screenshot 2023-08-22 at 21 07 29

**Comment by taylornowotny** [comment]
Looks real nice!

Some of these may need to change if we change the component names. I have made a note of it in my Name Change feedback doc https://www.notion.so/webstudiois/component-name-update-0a81ecde53cd4770bdbbbf248b167c4b

**Review by taylornowotny — APPROVED** [review]

**Review by kof — COMMENTED** [review]
We will change component labels separately

**Pull request closed without merging**